lib/questions.rb

```diff
@@ -187,12 +222,14 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  raise NameError if str_method == 'foobar'
 end
 
 # return true if the date is a uk bank holiday for 2014
 # the list of bank holidays is here:
 # https://www.gov.uk/bank-holidays
 def is_a_2014_bank_holiday?(date)
+  ['01/01/2014','18/04/2014','21/04/2014','05/05/2014','26/05/2014','25/08/2014','25/12/2014','26/12/2014' ].include? format_date_nicely(date)
```

Comment:
Line is too long. [142/80]<br>Space missing after comma.<br>Space inside square brackets detected.